thingiverse/ThingiverseApiClient.py

```diff
@@ -85,7 +85,7 @@ def get(self, query: str, page: int, on_finished: Callable[[List[JSONObject]], A
         :param on_finished: Callback method to receive the async result on.
         :param on_failed: Callback method to receive failed request on.
         """
-        url = "{}/{}?per_page={}&page={}".format(self._root_url, query, Settings.THINGIVERSE_API_PER_PAGE, page)
+        url = "{}/{}/?sort=relevent&per_page={}&page={}".format(self._root_url, query, Settings.THINGIVERSE_API_PER_PAGE, page)
```

Comment:
```suggestion
        url = "{}/{}/?sort=relevant&per_page={}&page={}".format(self._root_url, query, Settings.THINGIVERSE_API_PER_PAGE, page)
```

Reply:
This sounds strange but it has to be misspelled as "relevent" or it won't work. Another option if that bothers you is to use "popular". Other options are on this documentation: https://www.notion.so/Thingiverse-Search-API-f7ce7608d54d44f7a2b902a83194a8b2

Reply:
wow, thanks for clarifying. Thingiverse's API is a mess lately...

Reply:
Wait... What?

Reply:
They probably outsourced the whole thing to a non-native English country :P